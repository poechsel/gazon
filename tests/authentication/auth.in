ls
login u1
ls
pass p1
login u1
pass wrong-pass
login u1
pass p1
whoami
ls
login u2
pass p2
whoami
logout
whoami
ls
