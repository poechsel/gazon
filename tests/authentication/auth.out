Error: access denied.
Error: access denied.
Error: access denied.
Error: authentication failed.
u1
total 0
u2
Error: access denied.
Error: access denied.
